<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="GameWidget">
    <property name="geometry">
     <rect>
      <x>-300</x>
      <y>-230</y>
      <width>1611</width>
      <height>1171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="OpenMenu">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QPushButton" name="StartButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>340</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExitButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>470</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HTPButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>410</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>How To Play</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>250</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Zork: Murder Mystery!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>590</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Created by: Andrew Kelly &amp;amp; Conor McNelis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="HowToPlay">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(204, 142, 255);</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>230</y>
        <width>461</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-In this game you must solve the mystery behind &lt;/p&gt;&lt;p&gt;the murder at -SAMPLE NAME- mansion&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;-You have a limited amount of time, indicated by&lt;/p&gt;&lt;p&gt; the number of turns on your screen, to investigate&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;-The game is simple. pick a room, look for clues&lt;/p&gt;&lt;p&gt;, interrogate witnesses and at the end of the night, &lt;/p&gt;&lt;p&gt;select who you think the culprit is.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;-Your inventory is displayed on screen, &lt;/p&gt;&lt;p&gt;along with a helpful notepad to write down your theories&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;-Multiple playthroughs are encouraged to fully explore the game!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReturnButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>370</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="MainGame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <widget class="QPlainTextEdit" name="notes">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>230</y>
        <width>181</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="plainText">
       <string>Enter your notes here</string>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Descriptions">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>240</y>
        <width>261</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="GoNorthButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>550</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Go North</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GoWestButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>580</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Go West</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GoEastButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>580</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Go East</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GoSouthButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>610</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Go South</string>
      </property>
     </widget>
     <widget class="QComboBox" name="ItemList">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>550</y>
        <width>65</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExamineButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>530</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Examine</string>
      </property>
     </widget>
     <widget class="QComboBox" name="InventoryList">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>500</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <item>
       <property name="text">
        <string>Inventory</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="CharacterList">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>600</y>
        <width>65</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TalkButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>580</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Talk</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>479</x>
        <y>551</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Take</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OpenMapButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>500</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Open Map</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="MainLobby">
     <widget class="QLabel" name="DialogueBox">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>230</y>
        <width>291</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="ProfilePic">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>230</y>
        <width>161</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>480</y>
        <width>361</width>
        <height>161</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Where were you at 9:48pm last night?&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Did you ever have any issues with Mr Dead?&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Tell me a little bit about yourself&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="Option2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>550</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Option1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>520</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Option3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>580</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BackButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>610</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AccuseButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>510</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Accuse</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
